types/src/common.ts

```diff
@@ -26,7 +39,8 @@ export function JoyBTreeSet<V extends UInt>(valType: Constructor<V>): Constructo
       const encoded = new Array<Uint8Array>()
 
       if (!isBare) {
-        encoded.push(Compact.encodeU8a(this.size))
+        const sizeValue = new UInt(this.registry, this.size)
```

Comment:
I think this is incorrect, I'm getting 
```
Error: 1010: Invalid Transaction: Transaction has a bad signature
```

When submitting transactions with arguments that use a BTreeSet.
Will fix

Reply:
For some reason while coding I experienced that the `Compact` class  had "lost" the `encodeU8a` method, but it must have been some issue with my IDE TS plugin.. Anyway will reverted this change in 8b1505e9aa8b8834a65b909cbcf35326bfa04d08